src/shared/components/action-buttons/ToolsButton.tsx

```diff
@@ -32,60 +19,25 @@ const ToolsButton: FC<ToolsButtonProps> = ({
   children,
 }) => {
   const history = useHistory();
-  const dispatch = useDispatch();
 
   const entriesRef = useRef(selectedEntries);
   entriesRef.current = selectedEntries;
 
-  const [loading, setLoading] = useSafeState(false);
-
   const handleClick = useCallback(async () => {
-    setLoading(true);
-
     const entries = entriesRef.current;
 
-    try {
-      const sequences = await Promise.all(entries.map(getFASTAFromAccession));
-
-      if (entries !== entriesRef.current) {
-        // it means that by the time we get here, the selection has changed
-        return;
-      }
-      history.push(LocationToPath[location], {
-        parameters: { sequence: sequences.filter(Boolean).join('\n\n') },
-      });
-    } catch (err) {
-      if (entries !== entriesRef.current) {
-        // it means that by the time we get here, the selection has changed
-        return;
-      }
-
-      setLoading(false);
-
-      if (!(err instanceof Error)) {
-        return;
-      }
-      dispatch(
-        addMessage({
-          id: v1(),
-          content: err.message,
-          format: MessageFormat.POP_UP,
-          level: MessageLevel.FAILURE,
-        })
-      );
-    }
-  }, [dispatch, history, location, setLoading]);
-
-  useEffect(() => {
-    // reset loading state every time the selection changes
-    setLoading(false);
-  }, [selectedEntries, setLoading]);
+    history.push(LocationToPath[location], {
+      parameters: {
+        ids: entries,
+      },
+    });
+  }, [history, location]);
```

Comment:
No need to do that in a click handler now, you can just use a Link instead of a Button and pass what you're passing to history.push to the `to` prop

Reply:
A bit confused by the `ref`, was that to handle updates while the fetch hadn't finished?

Reply:
I think we still need this to be a Button because it can be disabled - also there might be styling divergences if we don't use the `Button` component.

Reply:
Had forgotten `Buttons` could be "anything" 😆

Reply:
In the end stayed with a regular `button` - there are styling side issues I didn't want to get into as part of this PR.

Reply:
Can you detail the styling issues you encountered?

Reply:
results page: vertical alignment , mouse over when "disabled"...
All are probably all addressable locally, but I imagine just the tip of the iceberg, so would need a more generic solution.

Reply:
Note that these issues are probably also present in all place where we combine both

Reply:
Yep ... except I don't think we currently use anchors within `.button-group`. This is why I'm saying this needs a generic solution (to be addressed globally).

Reply:
Entry feedback and add a publication, on the top of my mind

Reply:
![image](https://user-images.githubusercontent.com/3329765/145050443-b30a4968-bbb3-4628-97ba-8bdbe87b7ad2.png)
There's an issue there too...